### DISCUSSION TITLE
How to add HSTS header in Cockpit

### DISCUSSION THREAD

**Discussion by aziddam** [cockpit-project/cockpit#21441] [Q&A] [answered]
I'm trying to include the HSTS headers in cockpit but I cant get it to work. Any recommendations on how to get this done?

This is what I have in my cockpit.conf file 
[WebService]
Headers = Strict-Transport-Security: max-age=31536000; includeSubDomains; preload

**Comment by jelly** [comment, marked as answer]
Cockpit-ws has no `Headers` configuration option see the [configuration docs](https://cockpit-project.org/guide/latest/cockpit.conf.5.html). So my best advice is to setup a reverse proxy.
